packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -122,6 +122,11 @@ String getAarTaskFor(BuildInfo buildInfo) {
   return _taskFor('assembleAar', buildInfo);
 }
 
+@visibleForTesting
+const String androidX86DeprecationWarning =
+  'Support for Android x86 targets will be removed in the next stable release after 3.27. '
```

Comment:
Should this be `after 3.28`? The 3.28 branch has already been cut, so I we'll be removing support for anything in 3.28.

Reply:
The plan is to cherry pick this change so it's shipped with the upcoming stable release. Dart wants to remove x86 support for Dart 3.7.

Reply:
SGTM